lms/templates/courseware/accordion.html

```diff
@@ -32,6 +32,10 @@
                         span_start=HTML('<span class="sr">'),
                         span_end=HTML('</span>'),
                     ) if 'active' in section and section['active'] else ''}</p>
+
+                ## TEMP COMMENT
```

Comment:
See above comment about reverting this POC change from this PR.  A subsequent PR can do the full cleanup and change.